frontend/src/utils/types.ts

```diff
@@ -26,13 +26,30 @@ export type ActionList = Array<
 export type RelatedEntityObject = EntityNoExtraField | RelatedEntity;
 export type VideoObject = Video | VideoSerializerWithCriteria;
 
-// a poll that can be displayed by <PollSelector>
+/**
+ * An exhaustive list of route ids helping to enforce type checking in each
+ * place a routeId is required.
+ */
+export type routeId =
+  | 'home'
+  | 'recommendations'
+  | 'comparison'
+  | 'myComparisons'
+  | 'myRatedVideos'
+  | 'myRateLaterList'
+  | 'about';
```

Comment:
Maybe this type could be defined as an `enum`. 
It would allow us to write `RouteId.HOME` instead of `"home" as const` and possibly make things more explicit?

Reply:
Sure :+1:

Reply:
Fixed by 27e842c42e5a08b1b5e67c40e926272deb0f7284